internal/api/v1/application/export.go

```diff
@@ -201,7 +201,16 @@ func ExportToRegistry(c *gin.Context) apierror.APIErrors {
 
 	destinationURL, err := url.Parse(destination.URL)
 	if err != nil {
-		return apierror.InternalError(err)
+		if !strings.Contains(destination.URL, "://") {
+			// Run again with a scheme - This ensures that host and path are properly separated.
+			// Without a scheme the parser considers everything to be the path.
+			destinationURL, err = url.Parse("oci://" + destination.URL)
+			if err != nil {
+				return apierror.InternalError(err)
+			}
+		} else {
```

Comment:
This could fail providing an url like `/172-18-0-5.nip.io:5000/ak`: https://play.golang.com/p/5Xd3j37kwDj

The Host will be blank, and also the second parse with the Scheme will just add it but the Host will still be blank.

I would also change the comment adding this issue for future reference on why this was needed (comment unclear): https://github.com/golang/go/issues/18824

Reply:
Done with commit 17a8728948067881ba3fc7b593727e13b544b869.
Placed into separate function, and reworked for clarity.